webroot/src/components/SelectedStatePurchaseInformation/SelectedStatePurchaseInformation.less

```diff
@@ -0,0 +1,111 @@
+//
+//  SelectedStatePurchaseInformation.less
+//  CompactConnect
+//
+//  Created by InspiringApps on 11/12/2024.
+//
+.selected-state-purchase-info-container {
+    width: 27rem;
+    margin: 0.4rem 0;
+    padding: 1rem 1.6rem;
+    border-radius: 20px;
+    color: white;
+    background-color: @primaryColor;
+
+    @media @tabletWidth {
+        margin: 1rem 1rem;
+    }
+
+    .jurisprudence-modal {
+        :deep(.modal-container) {
+            max-width: 60rem;
+            padding: 2rem;
+
+            @media @tabletWidth {
+                padding: 4rem;
+            }
+
+            .jurisprudence-modal-content {
+                padding-top: 1rem;
+                overflow-y: hidden;
+                color: black;
+
+                .action-button-row {
+                    display: flex;
+                    flex-direction: row;
+                    justify-content: flex-end;
+                    margin-top: 4rem;
+                    margin-bottom: -3rem;
+
+                    .understand-button {
+                        margin-left: 1rem;
+                    }
+                }
+            }
+        }
+    }
+
+    .info-row {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        margin: 1rem 0;
+
+        :deep(.input-button-container) {
+            &.input-container {
+                margin: 0;
+
+                input {
+                    width: 4rem;
+                    height: 4rem;
+                    margin: -1rem -1.2rem 0 0;
+                    padding: 0;
+                    color: white;
+
+                    @media @tabletWidth {
+                        margin: 0 -1.2rem 0 0;
+                    }
+                }
+            }
+        }
+
+        &.sub-row {
+            color: rgba(255, 255, 255, 0.5);
```

Comment:
Ideally we stick with LESS variables. An interpolated form of `color: ~"rgba(@{white}, 0.5)";` might work. If not there is always the LESS function `color: fade(@white, 50%);`

High-level, transparent white on our eventual blue doesn't pass WCAG AA:
- https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF80&bcolor=2459A9

75% seems to be passing:
- https://webaim.org/resources/contrastchecker/?fcolor=FFFFFFBF&bcolor=2459A9